runner/steps.ts

```diff
@@ -161,7 +161,9 @@ function runBrowsers(context: Context) {
       // TODO(nevir): Better rationalize run-end and hook.
       await context.emitHook('cleanup');
 
-      return error;
+      if (error) {
+        throw error;
```

Comment:
Ok, but in our own code we must not assume that thrown objects / rejection values are Errors. It would be nice if that constraint were enforced, but it isn't.